kore/src/Kore/Rewriting/RewritingVariable.hs

```diff
@@ -109,13 +117,26 @@ instance From VariableName RewritingVariableName where
 
 instance FreshName RewritingVariableName
 
+instance AstWithLocation RewritingVariableName where
+    locationFromAst = locationFromAst . base . from @RewritingVariableName @VariableName
+
 type RewritingVariable = Variable RewritingVariableName
 
 mkElementConfigVariable
     :: ElementVariable VariableName
     -> ElementVariable RewritingVariableName
 mkElementConfigVariable = (fmap . fmap) ConfigVariableName
 
+configElementVariableFromId
+    :: Id -> Sort -> ElementVariable RewritingVariableName
+configElementVariableFromId identifier sort =
+    mkElementConfigVariable (mkElementVariable identifier sort)
+
+ruleElementVariableFromId
+    :: Id -> Sort -> ElementVariable RewritingVariableName
+ruleElementVariableFromId identifier sort =
+    mkElementRuleVariable (mkElementVariable identifier sort)
+
```

Comment:
Are these used outside of the test modules? They look more like test data builders to me, so they should probably be added to one of the test modules.

Reply:
Do you think `Test.Kore` is a good place?